api/sample_data/data.py

```diff
@@ -49,3 +45,34 @@ def reset_dataset(title, files, source, spacy_model):
         ratom_file.reported_total_messages = ratom_file.message_set.count()
         ratom_file.import_status = File.COMPLETE
         ratom_file.save()
+
+
+def sample_reset_all():
```

Comment:
Should we just add a 2nd "file" for Bill Rapp to the sample data? I think having a single "reset sample data" command just seems simpler and will be less confusing in the future.

Reply:
I dunno. There might be cases where you don't want Albert Meyers mixed in with Bill Rapp.  I thought this way the person could choose, or we could have three [Sample Data] accounts one that has both combined.

Reply:
Okay seems fine!